environment.yml

```diff
@@ -16,3 +16,5 @@ dependencies:
   - xeus-sql
   - soci-postgresql
   - soci-sqlite
+  - pip:
+    - nbgitpuller  # nbgitpuller on conda-forge seems to be outdated so use pip
```

Comment:
AFAIK, our 2i2c image is using 0.10.* 
https://github.com/2i2c-org/2i2c-hubs-image/blob/main/requirements.txt#L26
We should probably:
1. Get latest nbgitpuller in conda-forge
2. Update nbgitpuller in our 2i2c image